src/registrar/tests/common.py

```diff
@@ -626,15 +626,11 @@ def _convertPublicContactToEpp(
         self, contact: PublicContact, disclose_email=False, createContact=True
     ):
         DF = common.DiscloseField
-        fields = {DF.FAX, DF.VOICE, DF.ADDR}
-
-        if not disclose_email:
-            fields.add(DF.EMAIL)
+        fields = {DF.EMAIL}
 
         di = common.Disclose(
-            flag=False,
+            flag=disclose_email,
```

Comment:
I think we should add  two tests for the _disclose_fields in the contacts test class and just check once for a security contact and once for a non security contact. That way if something changes down the road we have a test for this.

On visual inspection it all looks good

Reply:
@abroddrick Okay I added some tests. The one that checks for the disclosure is redundant but its good to have that test case out separated into its own thing, in case a change happens to that beefier one